terasoluna-gfw-common-libraries/terasoluna-gfw-string/src/main/java/org/terasoluna/gfw/common/fullhalf/DefaultFullHalf.java

```diff
@@ -394,7 +394,7 @@
  * </tr>
  * <tr>
  * <td>｝</td>
- * <td>}</td>
```

Comment:
Please keep current format in this class using `<excludes>` on `pom.xml` .

See http://code.revelc.net/formatter-maven-plugin/examples.html.